packages/zql/src/mutate/mutator-registry.ts

```diff
@@ -174,34 +173,79 @@ export function mustGetMutator(
 /**
  * Checks if a value is a MutatorRegistry.
  */
-export function isMutatorRegistry<S extends Schema, C>(
-  value: unknown,
-): value is MutatorRegistry<S, C, MutatorDefinitions<S, C>> {
+export function isMutatorRegistry<
+  MD extends AnyMutatorDefinitions,
+  TSchema extends Schema = DefaultSchema,
+>(value: unknown): value is MutatorRegistry<MD, TSchema> {
   return (
     typeof value === 'object' && value !== null && mutatorRegistryTag in value
   );
 }
 
+/**
+ * Creates a function that can be used to define mutators with a specific schema.
+ */
+export function defineMutatorsWithType<
```

Comment:
I'm having trouble understanding why introduce this type? It's only used in one place in a test... Is it test specific? If so should it be defined in the test?